src/locales/en-US/commands.json

```
@@ -802,6 +802,11 @@
     "commandUsage": "<text>",
     "missingSentence": "You have to give me some text to vaporwavefy!"
   },
+  "leet": {
+    "commandDescription": "Leetifies the t3xt.",
+    "commandUsage": "<--hard(optional)><text>",
```

We don't usually add flags to the usage. Maybe in the future we could add a specific object in the locale for flags

```suggestion
    "commandUsage": "<text>",
```